telemetry/notes.txt

```diff
@@ -0,0 +1,30 @@
+sudo python telemetry_record.py -t 3
+
+
+cut -d ' ' -f 2 ../data/filtered_stats/valids.txt > valid_originals.txt
```

Comment:
Note this is referring to replays, not the "originals". Should I change the naming here?